transport/network/src/ping.rs

```diff
@@ -73,14 +73,14 @@ pub type PingQueryResult = Result<(std::time::Duration, String)>;
 
 /// Helper object allowing to send a ping query as a wrapped channel combination
 /// that can be filled up on the transport part and awaited locally by the `Pinger`.
-#[derive(Debug)]
+#[derive(Debug, Clone)]
 pub struct PingQueryReplier {
-    notifier: futures::channel::oneshot::Sender<PingQueryResult>,
+    notifier: UnboundedSender<PingQueryResult>,
```

Comment:
Shouldn't this be bounded(1)?

Reply:
I need the `unbounded_send` that does not need to be awaited, so I don't have to change a bunch of APIs to be suddenly `async`.